docs/features/mathtype/README.md

```diff
@@ -137,7 +137,7 @@ If you visit a page using MathType with your mobile device, the handwriting inte
 
 ## Input and Output
 
-MathType is based upon standards like MathML for internal representation and the PNG image format for displaying formulas. It can also handle other formats like LaTeX, Flash, SVG and EPS.
```

Comment:
I'm not sure if Flash should be removed here as Mathtype seems to still support this format https://docs.wiris.com/en/mathtype/mathtype_web/sdk/flash-package